docs/core/tools/dotnet-install-script.md

```diff
@@ -138,6 +138,14 @@ macOS/Linux:
 
 `./dotnet-install.sh --shared-runtime --version 1.1.0`
 
+#### Windows 'obtain-script and install' one-liner example
```

Comment:
This should follow the same template as the other examples above. For example, they are not in sub-headings. Something like this:

Obtain script and install .NET Core CLI one-liner:

Windows:

`@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&([scriptblock]::Create((Invoke-WebRequest -useb 'https://dot.net/v1/dotnet-install.ps1'))) <additional install-script args>"`

macOS/Linux:

`curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin <additional install-script args>`

Reply:
done...